.github/ISSUE_TEMPLATE/BUG_REPORT.md

```diff
@@ -45,4 +45,5 @@ Search open/closed issues before submitting since someone might have asked the s
 | ----------------- | ---------- |
 | Python            |
```

Comment:
Could you reorder this (and the table in the question template)? Order should be:

1. tamr-unify-client
2. Tamr Unify server
3. Python
4. Operating System

That way its sorted by most likely root cause.

Reply:
oop just saw this. Yes I can reorder it.